lib/admin/WP_Auth0_Admin.php

```diff
@@ -248,9 +248,12 @@ public function admin_enqueue() {
 		wp_enqueue_style( 'media' );
 	}
 
-	// TODO: Deprecate, not used
+	/**
+	 * @deprecated 3.6.0 - Handled empty auth0_app_token notification, not needed
```

Comment:
"not needed" sounds like an internal note. If this is user facing I'd use "This method is no longer required since..".